docs/deployment.md

```diff
@@ -4,8 +4,9 @@
 The following guides can help with the deployment to public cloud providers:
 * [DigitalOcean](digital-ocean.md)
 * [Heroku](heroku.md)
+* [AWS](aws.md)
```

Comment:
shell we sort these alphabetically?

Reply:
Sure, I pushed an update that fixes this.